bundles/org.openhab.binding.verisure/src/main/java/org/openhab/binding/verisure/internal/VerisureSession.java

```diff
@@ -528,6 +600,7 @@ private synchronized boolean logIn() {
                 store.get(authUri).forEach(cookie -> {
                     store.remove(authUri, cookie);
                 });
+                httpClient.setFollowRedirects(false);
```

Comment:
This is not allowed on the shared HTTP client. I believe you can also set this on `Request` using `.followRedirects(false)` so you won't have to create a dedicated client.

Reply:
Fixed!